### ISSUE TITLE
ProxyConnector doesn't always set correct request line

### ISSUE THREAD

**Issue by strayer** [aio-libs/aiohttp#368]
I tried using aiohttp with a HTTP proxy and ran into several problems. Most proxies simply broke and told me the client made a malformed request, some other notified me that I need to enable transparent proxy support.

I _think_ this is because the correct request line is only set when the first connection to the proxy is made.

The relevant code to update the request line for the proxy should be here: https://github.com/KeepSafe/aiohttp/blob/master/aiohttp/connector.py#L523

`_create_connection` seems to be called in `BaseConnector.connect`, which is only used in `ClientSession.request`, if I understand the code correctly.

If you look at the source of `BaseConnector.connect` you can see that `_create_connection` is only called as long as there is no existing connection - thus request that reuse an existing connection don't get an updated request line and will fail, as long as the HTTP proxy doesn't have transparent proxy support.

This also happens when reusing the same ProxyConnector for multiple requests, even if not using a Session object, since the connection to the proxy is stored in the Connector.

**Comment by asvetlov** [comment]
Hmm. Would you try `ProxyConnector(proxy_addr, force_close=True)`?
I'm inclined to use `force_close=True` as default value for `ProxyConnector`.

**Comment by strayer** [comment]
Seems to work... using a request session like this:

```
PROXY_CONNECTOR = aiohttp.ProxyConnector(proxy="http://127.0.0.1:12345", force_close=True)
SESSION = aiohttp.ClientSession(connector=PROXY_CONNECTOR)

SESSION.get(...)
```

**Comment by asvetlov** [comment]
Fixed in 792b980 -- `force_close=True` by default for `ProxyConnector`
